src/discussions/comments/comment/ResponseEditor.jsx

```diff
@@ -30,11 +36,13 @@ function ResponseEditor({
         />
       </div>
     ) : (
-      <div className={classNames({ 'mb-4': addWrappingDiv }, 'actions d-flex')}>
-        <Button variant="primary" className="px-2.5 py-2" onClick={() => setAddingResponse(true)}>
-          {intl.formatMessage(messages.addResponse)}
-        </Button>
-      </div>
+      !inBlackoutDateRange(blackoutDateRange) ? (
+        <div className={classNames({ 'mb-4': addWrappingDiv }, 'actions d-flex')}>
+          <Button variant="primary" className="px-2.5 py-2" onClick={() => setAddingResponse(true)}>
+            {intl.formatMessage(messages.addResponse)}
+          </Button>
+        </div>
+      ) : null
```

Comment:
nit:
`!inBlackoutDateRange(blackoutDateRange) && (
        <div className={classNames({ 'mb-4': addWrappingDiv }, 'actions d-flex')}>
          <Button variant="primary" className="px-2.5 py-2" onClick={() => setAddingResponse(true)}>
            {intl.formatMessage(messages.addResponse)}
          </Button>
        </div>
      ) `

after that, you will not need to comment eslint. ` // eslint-disable-next-line no-nested-ternary`